tests/utils.py

```diff
@@ -13,3 +14,46 @@ def wrapper(*args, **kwargs):
             return fn(*args, **kwargs)
         return wrapper
     return decorator
+
+
+def assert_logs(log_action, node_key, index=-1):
+    """A decorator to ensure a log is added during a unit test.
+    :param str log_action: NodeLog action
+    :param str node_key: key to get Node instance from self
+    :param int index: list index of log to check against
+
+    Example usage:
+    @assert_logs(NodeLog.UPDATED_FIELDS, 'node')
+    def test_update_node(self):
+        self.node.update({'title': 'New Title'}, auth=self.auth)
+
+    TODO: extend this decorator to check log param correctness?
```

Comment:
+1. Might want to add this to regression test https://github.com/CenterForOpenScience/osf.io/pull/4235#issuecomment-140117058